### ISSUE TITLE
An ADA issue: «Headers of footer columns have an insufficient color contrast»

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#8]
![01](https://user-images.githubusercontent.com/83204/46415840-fb72a480-c747-11e8-8a2a-0696c08dfab5.png)

**Comment by dmitry-fedyuk** [comment]
The parent task: [An ADA issue: «7. Insufficient color contrast»](https://github.com/kitchencollection-com/core/issues/3).

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.0.6](https://github.com/kitchencollection-com/core/releases/tag/0.0.6).
